gazette/css/blocks.css

```diff
@@ -194,6 +194,12 @@ p.has-drop-cap:not(:focus)::first-letter {
 	margin-bottom: 30px;
 }
 
+/* Newspack Blog Posts */
+
+.wpnbha p .more-link {
```

Comment:
I'm not sure what `.wpnbha` is but it seems you're targeting the newspack block perhaps better targeted via `.wp-block-newspack-blocks-homepage-articles` instead.

Reply:
I guess `.wpnbha` is just shorthand for the longer class name.  I would argue that the full class name of the element should be used in the theme CSS to avoid confusion for somebody looking at the code in the future.

Reply:
Sorry, I haven't been able to jump into this due to other duties. `.wpnbha` is a shorthand class. I've seen in used in the past in other themes (and have used it on other PRs) but I agree it isn't the most descriptive class name.